### DISCUSSION TITLE
how to chat?

### DISCUSSION THREAD

**Discussion by Nekto348** [PrismarineJS/mineflayer#2638] [Q&A] [answered]
tell me please how to make a bot write into chat that I write in the console

**Comment by u9g** [comment]
bot.chat -> write to chat
console.log -> write to console

**Reply by Nekto348** [reply]
That I know. But I need the text entered into the console to be displayed in the chat on the server.

**Comment by ghost** [comment, marked as answer]
Read the docs. Use readline or repl module.

**Comment by Grammf** [comment]
```
    rl.on('line', function (line) {
        bot.chat(line)
        rl.prompt();
    })
```
  Fast, simple and bad one i use
